test/functional/feature_fee_estimation.py

```diff
@@ -382,6 +388,41 @@ def test_acceptstalefeeestimates_option(self):
         self.start_node(0,extra_args=["-acceptstalefeeestimates"])
         assert_equal(self.nodes[0].estimatesmartfee(1)["feerate"], fee_rate)
 
+    def clear_estimates(self):
+        self.log.info("Restarting node with fresh estimation")
+        self.stop_node(0)
+        fee_dat = os.path.join(self.nodes[0].chain_path, "fee_estimates.dat")
```

Comment:
nit: if you retouch `Pathlib` lets you use the `/` operator to create child paths:

```suggestion
        fee_dat = self.nodes[0].chain_path / "fee_estimates.dat"
```

Reply:
fixed.
thanks